test/source/TestUtility.cpp

```diff
@@ -543,6 +554,20 @@ static int TestUtilityExchange()
 		EATEST_VERIFY(a == "EA Standard Template Library");
 	}
 
+	// Issue #82; construct pair using single move constructor
+	#if EASTL_MOVE_SEMANTICS_ENABLED
+	{
+		int i1 = 1;
+		TestPairSingleMoveConstructor<int> test1;
+		test1.test(eastl::move(i1));
+
+		int i2a = 2;
+		int & i2b = i2a;
+		TestPairSingleMoveConstructor<int&> test2;
+		test1.test(eastl::move(i2b));
```

Comment:
I'd agree that this isn't a useful test. What does it mean to "move" an object  into a reference.  If it even compiled, it would have to create temporaries for the reference to bind to which would be terminated leaving a dangling reference.

I think its fine to remove that test.